data/consent-page.js

```diff
@@ -0,0 +1,26 @@
+const studyValues = [
+  'studyName',
+  'addonName',
+  'duration',
+  'authors',
+];
+
+studyValues.forEach(property => {
+  const elems = document.getElementsByClassName(property);
+  [].forEach.call(elems, elem => {
+    elem.innerText = self.options[property];
```

Comment:
This is neat, but I wonder if want to do something fancier? On the study addons @casebenton worked on, we ended up using [yo-yo](https://www.npmjs.com/package/yo-yo), and it [worked out nicely[(https://github.com/mozilla/security-advisor-shield-study/blob/master/data/recommendation/notify.js#L46-L70) to get a more direct way of writing the html with variables in it.

Do you want to do something like this? Or would you rather go with the more separated way?

NB: yo-yo requires bringing in a very simple webpack config, since we would need to be able to import npm modules. But I think we'll want to be able to do that anyways.